doc/changes/unreleased.md

```diff
@@ -7,7 +7,8 @@
 
 ## ⚒️ Refactorings
 
-* Reformated entire code base with `black` and `isort`
+* Reformatted entire code base with `black` and `isort`
+* Modified `ExaConnection` so that default is encryption with strict certification validation
```

Comment:
not refactoring -> feature
add comment at top with breaking change and move to version 1.
-> see guidelines for